.cicd/docker/ubuntu-16.04.dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get upgrade -y \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git automake \
   libbz2-dev libssl-dev doxygen graphviz libgmp3-dev autotools-dev libicu-dev \
   python2.7 python2.7-dev python3 python3-dev autoconf libtool curl zlib1g-dev \
-  sudo ruby libusb-1.0-0-dev libcurl4-gnutls-dev pkg-config
+  sudo ruby libusb-1.0-0-dev libcurl4-gnutls-dev pkg-config apt-transport-https
```

Comment:
Is the new dependency for `buildkite-agent`?

Reply:
Yep. But it won't install when it's in the buildkite install section (I'm not sure why; it gets completely ignored even with a pruned environment/no cache)